astroplan/scheduling.py

```diff
@@ -508,8 +508,10 @@ def __init__(self, constraints, observer, transitioner=None,
         self.constraints = constraints
         self.observer = observer
         self.transitioner = transitioner
-        if not isinstance(self.transitioner, Transitioner):
-            raise ValueError("A Transitioner is required")
+
+        if not hasattr(self.transitioner, '__call__'):
```

Comment:
hmm, I believe `callable()` is the usual way to do this: https://docs.python.org/3/library/functions.html#callable